solutions/USACO dec18-bucketlist.mdx

```diff
@@ -41,3 +41,48 @@ file_out.close()
 </PySection>
 
 </LanguageSection>
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>  
+#include <cstdio>
+using namespace std;
+int event[1001]; // declaring the array globaly to intialize all the elements to 0
+int main() {
+	freopen("blist.in", "r", stdin);
+	freopen("blist.out", "w", stdout); 
+	int maxtime = 1000;
+	int n; cin >> n;
+		
+		
+	for (int i = 0; i < n; i++) { // iterating through n lines to input the data in
+		int start; cin >> start; 
+		int end; cin >> end;
+		int amt; cin >> amt;
+		event[start] += amt; 
+		// simulating the events by adding/subtracting the amount from si and ti
+		event[end] -= amt; // ^
```

Comment:
I feel like this should still be explained a bit more (either through more comments or through a brief editorial or both). I don't think Bronze people are familiar with this "prefix sum" ish sweep technique. What an "event" is isn't explained anywhere, for example